lib/routes/routeBackbeat.js

```diff
@@ -655,6 +658,12 @@ function routeBackbeat(clientIP, request, response, log) {
         });
         return responseJSONBody(errors.MethodNotAllowed, null, response, log);
     }
+    log.addDefaultFields({
```

Comment:
minor: if you do this just before `if (invalidRequest)` you might also be able to simplify the `invalid request` log call.

Reply:
It might also help to swap the invalid request boolean to a valid request. Positive booleans tend to be a little easier to follow the logic for.

Reply:
I agree, but I think changing logic code is a little out of scope for a log line ticket